docs/tasks/argo.rst

```diff
@@ -0,0 +1,253 @@
+Argo CRDs and setup their resources.
+Download the minimal manifest file from their Examples_ page.
+
+On downloading this yaml file, you may configure the managed namespaces
+as per your use case.
+You may read more about managed namespaces here - ManagedNamespaces_
+
+In this tutorial, the minimal manifest will be used as it is.
+
+.. code-block:: bash
+
+  kubectl apply -f quick-start-minimal.yaml -n argo
+
+Port-forward to view the Argo UI:
+
+.. code-block:: bash
+
+  kubectl -n argo port-forward deployment/argo-server 2746:2746
+
+Open a web browser and navigate to ``localhost:2746``
+
+.. image:: img/argo-default-ui.png
+
+Step 2 - Setup a sample application to backup.
+``````````````````````````````````````````````````
+
+Here, you will reference the MySQL_ example from Kanister.
+Install the Chart and setup MySQL in the ``mysql-test`` namespace.
+Make sure that the application is integrated with Kanister and the following
+steps are completed.
+
+1. A Profile CR is created for the ``mysql-test`` namespace.
+2. A MySQL blueprint is created in the ``kanister`` namespace.
+3. Check the name of the blueprint, stateful set, profile and
+   secrets for you application for the next step.
+
+Step 3 - Creating a Cron Workflow
+````````````````````````````````````````````
+
+Now a Cron Workflow will be created that will automate the creation of an actionset
+to backup our MySQL application. You will need to modify the names of the blueprint,
+stateful set, profile and secrets as per the ones you have created.
+Modify the last line of the following yaml file as follows.
+
+.. code-block:: bash
+
+  kanctl create actionset --action backup --namespace kanister --blueprint [YOUR_BLUEPRINT_NAME] --statefulset [YOUR_STATEFULSET] --profile [YOUR_PROFILE_NAME] --secrets [YOUR_SECRETS_NAME]
+
+Then execute -
+
+.. code-block:: bash
+
+  cat<<EOF >> mysql-cron-wf.yaml
+  apiVersion: argoproj.io/v1alpha1
+  kind: CronWorkflow
+  metadata:
+    name: mysql-cron-wf
+  spec:
+    schedule: "*/5 * * * *"
+    concurrencyPolicy: "Replace"
+    workflowSpec:
+      entrypoint: automate-actionset
+      templates:
+        - name: automate-actionset
+          container:
+            image: ghcr.io/kanisterio/kanister-tools:v9.99.9-dev
+            command:
+              - /bin/bash
+              - -c
+              - |
+                microdnf install tar
+                curl -LO https://github.com/kanisterio/kanister/releases/download/0.80.0/kanister_0.80.0_linux_amd64.tar.gz
+                tar -C /usr/local/bin -xvf kanister_0.80.0_linux_amd64.tar.gz
+                kanctl create actionset --action backup --namespace kanister --blueprint mysql-blueprint --statefulset mysql-test/mysql-release --profile mysql-test/s3-profile-gd4kx --secrets mysql=mysql-test/mysql-release
+  EOF
+
+Note - Here, the cron-job is scheduled to run every 5 minutes. You may schedule
+it to run as per your requirements.
```

Comment:
Can we use the way .rst recommends us to specify notes.

Reply:
we should also specify what does it mean to run the job every 5 minutes.
We should clarify that it means the actionset is going to get created every 5 minutes, that would result in backup being done every 5 minutes.
Maybe better version of above statement.

Reply:
Done.